src/Symfony/Component/Workflow/Debug/TraceableWorkflow.php

```diff
@@ -30,6 +30,7 @@ class TraceableWorkflow implements WorkflowInterface
     public function __construct(
         private readonly WorkflowInterface $workflow,
         private readonly Stopwatch $stopwatch,
+        public bool $enabled = true,
```

Comment:
I don't understand, the value will always be `true`?

Reply:
It's true by default to not change the default behavior, and set to false with wiring (see changes on compiler passes / bundle extensions).

Reply:
I saw you did it for some services, but I did not see it for the workflow :/ Not sure If you forgot it, or I'm blind 😅

Reply:
fixed